subsys/metrics/src/main/java/org/commonjava/indy/metrics/zabbix/reporter/IndyZabbixReporter.java

```diff
@@ -302,7 +301,7 @@ else if ( logger.isDebugEnabled() )
                 logger.info( "report metrics to zabbix success. " + senderResult );
             }
         }
-        catch ( IOException e )
+        catch ( Exception e )
```

Comment:
Because of here(https://github.com/Commonjava/indy/blob/29fa67ea8724c43f28ee6d5fbaba321e34749c81/subsys/metrics/src/main/java/org/commonjava/indy/metrics/zabbix/sender/IndyZabbixSender.java#L359-L359), I inject @IndyHttpProvider and throw IndyHttpException, so I must change IOException to Exception